Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -503,7 +504,7 @@ static public bool isRetryableHTTPCode(int statusCode, bool forceRetryOn404)
         {
             if (forceRetryOn404 && statusCode == 404)
                 return true;
-            return (500 <= statusCode) && (statusCode < 600) ||
+            return 500 <= statusCode && statusCode < 600 ||

```

Comment:
maybe `(500 <= statusCode && statusCode < 600)` would be more readable?

Reply:
Right, done